tests/benchmarks/multicore/idle_with_pwm/remote/boards/nrf54h20dk_nrf54h20_cpurad.overlay

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	aliases {
+		pwm-led0 = &pwm_gpio;
+	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+		/*
+		 * LEDs are connected to GPIO Port 9 - pins 0-3. There is no valid hardware
+		 * configuration to pass PWM signal on pis 0 and 1. First valid config is P9.2.
+		 * Signal on PWM130's channel 0 can be passed directly on GPIO Port 9 pin 2.
+		 * PWM131 can't access GPIO Port 9, thus use GPIO Port 0, pin 7.
+		 */
+		pwm_gpio: pwm_gpio {
+			pwms = <&pwm131 0 PWM_MSEC(20) PWM_POLARITY_NORMAL>;
+		};
+	};
+};
+
+&pinctrl {
+	/omit-if-no-ref/ pwm131_default: pwm131_default {
+		group1 {
+			psels = <NRF_PSEL(PWM_OUT0, 7, 0)>;
```

Comment:
ToDo: Confirm PWM signal is present on that GPIO.

Reply:
Wrong GPIO, shall be NRF_PSEL(PWM_OUT0, 0, 7)
PWM signal on P0.07 was confirmed with Saleae.